app/selfserve/module/Olcs/src/Service/Cookie/AcceptAllSetCookieGenerator.php

```diff
@@ -21,11 +21,11 @@ public function __construct(private readonly PreferencesSetCookieGenerator $pref
      *
      * @return SetCookie
      */
-    public function generate()
+    public function generate(?bool $state = true)
```

Comment:
This type hint suggests a null value is allowed, but it looks to always be required? Also can move the return type out of the docblock and into the code

```suggestion
    public function generate(bool $state = true): SetCookie
```